src/Orchard.Web/Modules/Upgrade/Controllers/TaxonomyController.cs

```diff
@@ -79,22 +79,26 @@ public class TaxonomyController : Controller {
             return RedirectToAction("Index");
         }
 
+        private const int BATCH = 50;
```

Comment:
Should be "BatchSize". Btw class-level variables should be at the top before the ctor, so they are more visible.